lib/tool_belt/config.rb

```diff
@@ -8,18 +8,37 @@ class Config
 
     def initialize(config_file, version=nil)
       configs = YAML.load_file(config_file)
-      valid?(configs)
-      configs = add_release(configs, version)
+      validate_bugzilla?(configs)
+      configs = add_bugzilla_release(configs, version)
+      #validate_redmine?(configs)
+      #configs = add_release(configs, version)
```

Comment:
There would essentially be two types of configuration files now -- Redmine and bugzilla that are formatted slightly differently and we need to be able to discern which is which and validate it properly.